PendingReleaseNotes

```diff
@@ -133,6 +133,15 @@
 * RGW: Added support for the `RestrictPublicBuckets` property of the S3 `PublicAccessBlock`
   configuration.
 
+* CephFS: `ceph fs purge status` command now allows users to check the amount of
+  progress made by threads in purging deleted subvolumes by printing statistics
+  of the progress. The statistics includes how many subvolumes and files as well
```

Comment:
s/by printing statistics of the progress//
s/includes/include/

Reply:
> s/by printing statistics of the progress//

I would like to not make this replacement because this phrase adds the emphasis that the progress is shown by printing statistics, in contrast to the other release note where the progress is shown by printing a progress bar.